src/components/Footer/Footer.js

```diff
@@ -0,0 +1,115 @@
+import {
+    Grid,
+    Container,
+    Button,
+    TextField,
+    Box,
+    Typography,
+    AppBar,
+    Toolbar,
+  } from "@mui/material";
+  
+  function Footer() {
+    return (
+      <footer>
+        <Box px={{ sm: 5 }} py={{ sm: 5 }}>
+          <Container maxWidth="lg">
+            <Grid container spacing={3}>
+              <Grid item xs={12} sm={4}>
+                <Typography variant="h2" sx={{ paddingBottom: "29px" }}>
```

Comment:
One more suggestion.
Try to create a component (for instance LinksGroup) that will take name, links as props.
For this part of the footer.
![image](https://user-images.githubusercontent.com/61732059/195646154-eba672f0-3d68-459d-ac3e-9bdb5ec5331c.png)

Use these props later in your component.
Firs will be used for the title of your list and the second will used to render a list of links
links.map(link=><Typography ....)

![image](https://user-images.githubusercontent.com/61732059/195647655-8793dac7-6b83-494a-9bf3-320cbf83ba67.png)


Also, it will be better to change it to links, not just Typography.